indico/web/templates/forms/markdown_widget.html

```diff
@@ -1,7 +1,120 @@
 {% extends 'forms/base_widget.html' %}
 
 {% block html %}
-    <textarea name="{{ field.name }}" rows="{{ rows }}" {{ input_args|html_params }}>
-        {{- field._value() -}}
-    </textarea>
+    {% if field.use_wmd %}
+        <div data-field-id="{{ field.orig_id }}">
+            <div class="wmd-panel">
+                <div id="wmd-button-bar-f_{{ field.orig_id }}" class="wmd-button-bar"></div>
+                <textarea id="{{ field.id }}"
+                          name="{{ field.name }}"
+                          rows="{{ rows }}"
+                          class="wmd-input"
+                          style="width: 100%;"
+                          {% if not field.use_mathjax %}data-no-mathjax{% endif %}
+                          {{ input_args|html_params }}>
+                    {{- field._value() -}}
+                </textarea>
+            </div>
+            <div class="md-preview-wrapper edit">
+                <div id="wmd-preview-f_{{ field.orig_id }}" class="wmd-preview"></div>
+                <div class="icon-arrow-up instructions">
+                    {% if field.use_mathjax %}
+                        {% trans markdown='<a href="#" class="markdown-info">Markdown</a>'|safe,
+                                 latex='<a href="#" class="latex-info">LaTeX</a>'|safe -%}
+                            You can use {{ markdown }} and {{ latex }} formulae.
```

Comment:
I'd say "**mathematical** formulae", since it's currently nor very clear what that means. Some people even added actual LaTeX directives.